.travis.yml

```diff
@@ -31,6 +31,7 @@ install:
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then gem install danger danger-junit; fi'
 
 script:
+  - docker-compose exec web bash -c "apt install -y phantomjs"
```

Comment:
Why can't we place this installation in our Dockerfile instead of executing this every build run. It would make easier when we accomplish this https://github.com/publiclab/mapknitter/issues/598

Reply:
Thanks for the review!

This pulls `libqt5webkit5` and hundreds of megabytes of libraries, and is not needed in production. At the moment it is saving some time and storage for building the container in production and making image more manageable for debugging it.

Reply:
Anyway we can revisit when we do start using docker images (exciting!). For now this should help trim testing and simplify deployment to production (where phantomjs isn't needed).